### DISCUSSION TITLE
Compatibility tests

### DISCUSSION THREAD

**Discussion by alisaifee** [dragonflydb/dragonfly#120] [General]
I've added a couple of integration tests in the CI for https://github.com/alisaifee/coredis at the moment covering:
- string operations
- hash operations
- list operations
- set operations
- sorted set operations

You can see them [here](https://github.com/alisaifee/coredis/runs/6785086475?check_suite_focus=true) for example.

I'll add pubsub once the resolution to #111 is available.

Would it be useful to reference this somewhere in the documentation?

**Comment by romange** [comment]
Definitely. We have "API readiness doc" linked from README. 
We could put the reference there, before all the APIs expansions.
